### PR TITLE
FE: Created Navbar and Nav Skeleton

### PR DESCRIPTION
**Pull request by dproc96**
**What it does**
- Add navigation bar to all pages
- Add links to the navbar for prospective pages
- Adds routes for prospective pages
- Adds mobile responsiveness for small and extra small screens
- Add Favicon and Document Name

**What to look for**
- Responsiveness
- Functioning navigation

**What to avoid**
- All routes show the Landing Page at the moment
- No indication for active page at this time
- All links are visible at all times

### CODE REVIEW

**Review by dahrk — APPROVED** [review]
Looks good; Follows material-ui specs for navbar. Very speedy :+1:

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    setAnchorEl(null);
+  };
+
+  const navOptions = {
+      classes,
+      anchorEl,
+      handleClick,
+      handleClose,
+      width: props.width
+  }
```
Comment (Braagaa) [inline client/src/components/Navbar.js:123]: Never knew about sending components parameters as an object. Nice!

**Comment by Braagaa** [comment]
I agree it looks good.

File: client/src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          id="menu"
+          anchorEl={anchorEl}
+          keepMounted
+          open={Boolean(anchorEl)}
+          onClose={handleClose}
+        >
+          {navLinks.map(({ title, path }) => (
+            <Link className={classes.linkText} to={path} key={title}>
+              <MenuItem onClick={handleClose}>{title}</MenuItem>
+            </Link>
```
Comment (aeciorc) [inline client/src/components/Navbar.js]: ReactRouter has a [NavLink](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md) component that would work well for this. It lets you apply styling to the link based on whether "to"  matches the current url

File: client/src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+  const navOptions = {
+      classes,
+      anchorEl,
+      handleClick,
+      handleClose,
+      width: props.width
+  }
+  return (
+    <AppBar position="static" className={classes.navbar}>
```
Comment (aeciorc) [inline client/src/components/Navbar.js:125]: You want the position to be "sticky" instead, so that it stays in place when scrolling down

**Review by aeciorc — APPROVED** [review]
Looks great! Can be merged after addressing the comments below:

File: client/src/components/Navbar.js (hunk 2 above)
Comment (dproc96) [reply client/src/components/Navbar.js]: Updated!

File: client/src/components/Navbar.js (hunk 3 above)
Comment (dproc96) [reply client/src/components/Navbar.js:125]: Updated in pull request #26 for Profile Settings page

**Pull request merged**